package.json

```diff
@@ -190,6 +190,110 @@
         "darwin",
         "linux"
       ]
+    },
+    {
+      "description": ".NET Core Debugger (Windows / x64)",
+      "url": "https://vsdebugger.azureedge.net/coreclr-debug-1-5-0/coreclr-debug-win7-x64.zip",
+      "installPath": ".debugger",
+      "runtimeIds": [
+        "win7-x64"
+      ]
+    },
+    {
+      "description": ".NET Core Debugger (OSX / x64)",
```

Comment:
I was basing it off omnisharp: "description": "OmniSharp (.NET Core - OSX / x64)",
I'll change it in both places.